miner.go

```diff
@@ -38,3 +38,29 @@ func ParseFile(fileName string, analytics *Analytics) {
 		fmt.Printf("Skipped %d non-JSON lines in '%s'.\n", skippedLines, fileName)
 	}
 }
+
+func ParseAndConvertFile(fileName string, converter *Converter) {
```

Comment:
I thought of this, but I didn't take the time to figure out how to create an interface in Go. I figured I would wait to see what you thought before I went that deep.